adapters/richaudience/richaudience.go

```diff
@@ -52,10 +52,18 @@ func (a *adapter) MakeRequests(request *openrtb2.BidRequest, requestInfo *adapte
 		}
 
 		if request.App != nil {
+
```

Comment:
Nitpick: Please remove the leading white space here and for the Site check.

Reply:
Done